### DISCUSSION TITLE
<c-t> to open git difftool does not do anything

### DISCUSSION THREAD

**Discussion by devployment** [jesseduffield/lazygit#3382] [Q&A] [answered]
According to [this file](https://github.com/jesseduffield/lazygit/blob/master/docs/keybindings/Keybindings_en.md) `<c-t>` should open `git difftool`. Trying to do so in the files pane on an selected file does nothing for me. 

Executing `git difftool` in the shell opens Beyond Compare (bc) just fine. Here's my `.gitconfig`.

Is there anything I'm missing here? Or a known issue related to this?

```
[core]
  autocrlf = input
  pager = delta

[interactive]
  diffFilter = delta --color-only

[delta]
  navigate = true    # use n and N to move between diff sections
  light = false      # set to true if you're in a terminal w/ a light background color (e.g. the default macOS terminal)
  feature = catppuccin-frappe
  side-by-side = true
  line-numbers = true

[merge]
  conflictstyle = diff3
  tool = bc

[diff]
  colorMoved = default
  tool = bc
  guitool = bc

[difftool]
  prompt = false

[mergetool "bc"]
  trustExitCode = true
```

**Comment by stefanhaller** [comment, marked as answer]
It's not released yet. The documentation describes the state of master. We hope to have a new release out very soon now.

**Reply by devployment** [reply]
Oh. Sorry. Missed that part. Thanks for the update.
